sql-cli/Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: help
+.DEFAULT_GOAL:= help
+SHELL := /bin/bash
+SYSTEM_PIP := pip
+
+
+clean:  ## Remove temporary files
+	@echo "Removing cached and temporary files from current directory"
+	@rm -rf logs
+	@find . -name "*.pyc" -delete
+	@find . -type d -name "__pycache__" -exec rm -rf {} +
+	@find . -name "*.sw[a-z]" -delete
+	@find . -type d -name "*.egg-info" -exec rm -rf {} +
+
+system:  ## Install system dependencies
+	@echo "Installing system dependencies using default pip"
+	@$(SYSTEM_PIP) install poetry
+	@$(SYSTEM_PIP) install pre-commit
+
+install:  ## Create Python virtualenv and install python dependencies
+	@poetry install
+
+virtualenv:  ## Activate project virtualenv
+	@echo "The virtual env is:" $(shell poetry env info -p)
+
+setup: system install  ## Setup a local development environment
+
+pre-commit:  # Run pre-commit
+	@git ls-files -- . | xargs pre-commit run --files
+
+flow:  # Build and run the latest version of flow
+
+test:  # Run tests
+	@poetry run pytest -s --cov --cov-branch --cov-report=term-missing
```

Comment:
Are we not using nox to run pytest?

Reply:
Nox is using this same command line at the moment..!

For local dev, if we can avoid yet another tool, I'd prefer - but if you feel strongly about having nox here, I can add it, @feluelle!

Reply:
I would say we should be consistent and use the same things in local dev and ci. If we use nox in CI, let's use it locally as well. If we can remove nox from ci, I am fine with this too.

Reply:
`sql-sdk` I have never heard of it. What is it? 😆